application/src/js/all/_table_with_fixed_header.js

```diff
@@ -11,25 +11,31 @@ function TableWithFixedHeader(outerTableElement) {
       innerTableElement = outerTableElement.querySelector('.table-container-inner')
       tableContainer = outerTableElement.querySelector('.table-container')
       tableElement = outerTableElement.querySelector('table')
+      tableCaption = outerTableElement.querySelector('caption')
 
       if (tableContainer) {
 
         tableHeader = tableElement.querySelector('thead')
 
         var fixedTableHeader = tableHeader.cloneNode(true)
+        var fixedTableCaption = tableCaption.cloneNode(true)
```

Comment:
Probably need to add a condition around this, as the `caption` is optional? (Unless we always want a caption, in which case we should probably remove the condition from the template)

Reply:
I don't think the caption is optional, pretty sure `header` is a required field (because it's the table title). Will check and remove the conditional, if true.